package.json

```diff
@@ -3,22 +3,28 @@
   "version": "1.0.0",
   "description": "This project was created to redesign the rbglive page (https://live.rbg.tum.de/). It uses HTML-Scraping to get the necessary information from the original page and then wraps it into a modern, dark-themed UI. Currently you can watch live streams and VoDs via the custom UI, Archives are not covered yet.",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "postinstall": "npm run purgeBootstrap",
+    "purgeBootstrap": "npx purgecss --css node_modules/bootstrap/dist/css/bootstrap.min.css --content \"tmpl/*\" --output node_modules/bootstrap/dist/css/bootstrap.purged.css --safelist collapsing show"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/TUM-Dev/RBGreaterAgain.git"
   },
   "author": "",
-  "license": "GNU3",
+  "license": "GPL-3.0-only",
   "bugs": {
     "url": "https://github.com/TUM-Dev/RBGreaterAgain/issues"
   },
   "homepage": "https://stream.tum.sexy",
   "dependencies": {
-    "@material-icons/font": "^1.0.6",
+    "aos": "^2.3.4",
+    "bootstrap": "^4.5.3",
+    "font-awesome": "^4.7.0",
     "hls.js": "^0.14.17",
-    "jquery": "^3.5.1",
-    "typeface-roboto": "^1.1.13"
```

Comment:
I guess you didn't like this font? :stuck_out_tongue:

Reply:
I just redid this completely and it looked okay with the default font. I can get this back in if you liked it.

Reply:
I don't care much about fonts :shrug: